script/Ciel/Day 13/13_01_01_CIEL13_5C.txt

```diff
@@ -233,7 +233,7 @@ You're going to die right here,
 [sha:4a779d45089e18efddaedc1f16bede9a620ed32e]{
 -- Page 673, Offset 35206.
 -- これって<希|レ><少|ア>かつ一度きりの美味しい機会だし」
-I mean, this kind of delectable opportunity comes only once a lifetime."
+I mean, this kind of delectable opportunity comes only once in a lifetime."
```

Comment:
I think original correct (comes only once a /year/lifetime/)

Reply:
preference to keep the 'in'

Reply:
Once in a lifetime is more common.
![image](https://user-images.githubusercontent.com/90473809/234786384-e073a214-ee88-417b-9b3e-02aaad1a0023.png)